fusor-ember-cli/app/controllers/deployment-new/satellite/configure-environment.js

```diff
@@ -7,7 +7,7 @@ export default Ember.Controller.extend(ConfigureEnvironmentMixin, {
 
   organizationTabRouteName: Ember.computed.alias("controllers.deployment-new.organizationTabRouteName"),
 
-  selectedOrganization: Ember.computed.alias("controllers.deployment-new.organization"),
+  selectedOrganization: Ember.computed.alias("controllers.deployment-new.model.organization"),
```

Comment:
@jwmatthews, the need for `model` is due to changing from `Ember.ObjectController` to `Ember.Controller` as `Ember.ObjectController` is being deprecated. ObjectController proxied the model and was convenient to have not need to write `model`. Now, we need to check if the attribute is originally defined on the controller or the model.